perception/compare_map_segmentation/include/compare_map_segmentation/voxel_based_approximate_compare_map_filter_nodelet.hpp

```diff
@@ -53,5 +53,5 @@ class VoxelBasedApproximateCompareMapFilterComponent : public pointcloud_preproc
 }  // namespace compare_map_segmentation
 
 // clang-format off
-#endif  // COMPARE_MAP_SEGMENTATION__VOXEL_BASED_APPROXIMATE_COMPARE_MAP_FILTER_NODELET_HPP_ // NOLINT
+#endif  // COMPARE_MAP_SEGMENTATION__VOXEL_BASED_APPROXIMATE_COMPARE_MAP_FILTER_NODELET_HPP_  // NOLINT
```

Comment:
@isamu-takagi Is this the intended behavior?

Reply:
Yes, ros-include-guard always inserts two spaces before double slashes.

Reply:
Okay, since it's consistent with these lines, I think it's fine. :+1: 
https://github.com/autowarefoundation/autoware.universe/blob/a1294c0f8241a4f07a30c18f85f6e6b507ffa3e2/perception/compare_map_segmentation/include/compare_map_segmentation/voxel_based_approximate_compare_map_filter_nodelet.hpp#L15-L16